src/components/Organizations/Detail/View.js

```diff
@@ -24,6 +27,14 @@ const MembersTable = styled.div`
   }
 `;
 
+const UpgradeNoticeWrapperDiv = styled.div`
+  cursor: pointer;
+  width: 22px;
+  span {
+    display: none !important;
+  }
+`;
+
```

Comment:
If I wrap the component into an styled one `styled(UpgradeNotice)` for some reason the tooltip doesn't shoew up...